src/server/pbsd_main.c

```diff
@@ -1436,13 +1393,6 @@ main(int argc, char **argv)
 	}
 	process_hooks(periodic_req, hook_msg, sizeof(hook_msg), pbs_python_set_interrupt);
 
-	/*
-	 * Make the scheduler (re)-read the configuration
-	 * and fairshare usage.
-	 */
-	(void)contact_sched(SCH_CONFIGURE, NULL, pbs_scheduler_addr, pbs_scheduler_port);
-	(void)contact_sched(SCH_SCHEDULE_NULL, NULL, pbs_scheduler_addr, pbs_scheduler_port);
-
```

Comment:
we are sending SCH_CONFIGURE as soon as sched connects to server, please see req_register_sched() in process_request.c, and despite that also fairshare test fails.

Reply:
If PTL test case is only the problem and it is looking for the message "Scheduler is reconfiguring" and it is not getting it because of the general default log_event_mask can we move the following statement after the function call set_validate_sched_attrs in case SCH_CONFIGURE: of schedule() function.
log_event(PBSEVENT_SCHED, PBS_EVENTCLASS_SCHED, LOG_INFO,
		"reconfigure", "Scheduler is reconfiguring");
If this is the last line of the above case then I think it should work. We can give a try.

If we do this as part of the function call set_validate_sched_attrs we fetch log_events from server.

Reply:
Yes @suresh-thelkar that's the case, and changing position log_event will work, but I have alternate solution, by default almost all log events (except high debug levels - aka DEBUG3 and DEBUG4) is added to log_event_lvl_priv in pbs_event.c but PBSEVENT_SCHED is not added to it and I am not sure why?
@bhroam Do you know why we have not added PBSEVENT_SCHED in log_event_lvl_priv (aka default log event mask)? Can we add it or not?

Reply:
@suresh-thelkar We don't want to send a SCH_CONFIGURE when the server starts.  There is a specific scheduler command which is sent (SCH_SCHEDULE_FIRST).  If there is something to be done when the server starts, it should be done there.  @hirenvadalia I can only guess at the answer to your question because it was done before my time.  My guess is that originally only the server and mom were written, and a shell of a scheduler.  Since the scheduler wasn't written, PBSEVENT_SCHED was either left out, or it was a mistake.  For 20+ years, the scheduler didn't use log_event(), so it didn't matter.  Now that the scheduler uses log_event(), we need to add PBSEVENT_SCHED into the default log mask.  I wouldn't add DEBUG2-4 though.

Reply:
@bhroam Thank for details, I will add PBSEVENT_SCHED in default log mask, however DEBUG2 is already part of default log mask, not sure why, but will keep that as it is.

On:
> @suresh-thelkar We don't want to send a SCH_CONFIGURE when the server starts. There is a specific scheduler command which is sent (SCH_SCHEDULE_FIRST). If there is something to be done when the server starts, it should be done there. 

@bhroam Are you sure that SCH_SCHEDULE_FIRST should go before SCH_CONFIGURE as soon as sched connects to server? (or earlier when server was connection to sched)
As what I see in mainline code (and which I retained in this PR also), is as soon as server starts the very first command it sends to sched is SCH_CONFIGURE -> SCH_SCHEDULE_NULL-> SCH_SCHEDULE_FIRST, this is the sequence goes to sched.
And in this PR changes, I removed SCH_SCHEDULE_NULL as it doesn't do anything in sched, sched just ignores it.

Reply:
Interesting, I didn't realize a SCH_CONFIGURE was sent.  I thought it was only a SCH_SCHEDULE_FIRST.  I'm not necessarily sure we need both, but I don't see any reason to change it right now if that's what is happening.  Knowing the SCH_CONFIGURE is sent, I probably can remove some code from SCH_SCHEDULE_FIRST.